### ISSUE TITLE
Audio mixer: modals don't work with keyboard

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#205]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/

Tab to Save or New button and activate 

Expected:

One of the buttons to have focus in the modal and it to trap tab focus until dismissed. Esc should close the modal too (basically should work like our library's modal)

Actual:

Modal button doesn't have focus (not sure where focus is). Pressing tab focuses things behind the modal.

Possible fix:

Might be possible to switch to our regular modal (although it is styled differently).
